spec/generators/install_generator_spec.rb

```diff
@@ -1,27 +1,33 @@
-# -*- encoding: utf-8 -*-
```

Comment:
Well obviously only if the file doesn't contain any non-ascii characters.. but yeah, that could be a worthwhile cleanup. Obviously I did here because I was touching the file. How can we tell whether the file contains any non-ascii characters (without reading it, which would take far too long)? I suppose you can just do `ruby <filename>` - the thing will crash immediately because of missing constants or whatever, but it proves the file can be read. So ok, there aren't too many files with this line, so I'll see about removing it from everywhere it's not needed.